graphql-dgs-webflux-starter/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+    api(project(":graphql-dgs-spring-boot-oss-autoconfigure"))
+    api(project(":graphql-dgs-spring-webflux-autoconfigure"))
+    api(project(":graphql-dgs-client"))
+    api(project(":graphql-error-types"))
+
+    implementation("org.springframework.boot:spring-boot-starter-webflux")
```

Comment:
So I need to add dependency in my project to `graphql-dgs-webflux-starter` and  `org.springframework.boot:spring-boot-starter-webflux`, am I right?
wouldn't it be better to mark is as `api` dependency? So after that  `graphql-dgs-webflux-starter` will be enough.

Reply:
done